packages/fuels-test-helpers/Cargo.toml

```diff
@@ -21,17 +21,13 @@ fuel-types = { workspace = true, features = ["random"] }
 fuels-accounts = { workspace = true, optional = true }
 fuels-core = { workspace = true }
 futures = { workspace = true }
-hex = { workspace = true, default-features = false, features = ["std", "serde"] }
 portpicker = { workspace = true }
 rand = { workspace = true, default-features = false }
-serde = { workspace = true, features = ["derive"] }
-serde_json = { workspace = true, features = ["raw_value"] }
-serde_with = { workspace = true }
 tempfile = { workspace = true, default-features = false }
 tokio = { workspace = true, default-features = false }
 which = { workspace = true, default-features = false }
 
 [features]
 default = ["fuels-accounts", "std"]
 std = ["fuels-accounts?/std", "fuels-core/std"]
-fuel-core-lib = ["fuel-core"]
+fuel-core-lib = []
```

Comment:
The original idea behind the feature is to speed up compilation by not having to compile the whole core.

Why is it necessary to always have the core dep present? Maybe you can use `fuel_core_chain_config` when configuring a snapshot for a binary?

Reply:
yup, was gonna try that next